OracleGoldenGate/runOracle.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
This file should be reused from the base container and not being duplicated. `runOracle.sh` is available inside your container as you have based the image on the database image.